### ISSUE TITLE
Javadocs don't tell all the truth about Mockito spies

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dlsgusrn7577/mockito#26]
```
Reported here: http://groups.google.com/group/mockito/t/3945fe1eca2954e7
```

Original issue reported on code.google.com by `szcze...@gmail.com` on 12 Oct 2008 at 8:23

**Comment by GoogleCodeExporter** [comment]
```
fixed in trunk
```

Original comment by `szcze...@gmail.com` on 12 Oct 2008 at 8:24
- Changed state: **Verified**

**Comment by GoogleCodeExporter** [comment]
Original comment by `szcze...@gmail.com` on 19 Apr 2009 at 7:40
- Added labels: **Milestone-Pre1.7**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 2044ae079e.
```

Original comment by `szcze...@gmail.com` on 7 Oct 2010 at 5:19
- Changed state: **Fixed**
